server/aws-lsp-codewhisperer/src/shared/localProjectContextController.ts

```diff
@@ -154,7 +158,9 @@ export class LocalProjectContextController {
             if (vecLib) {
                 this._vecLib = await vecLib.start(LIBRARY_DIR, this.clientName, this.indexCacheDirPath)
                 if (enableIndexing) {
-                    void this.buildIndex()
+                    void this.buildIndex('all')
```

Comment:
the errors inside this void buildIndex() will not be caught by the try catch.

You can do buildIndex().catch((e) => {} )

Reply:
Please log the error though